js/levels.js

```diff
@@ -33,4 +32,15 @@ function createTestLevel()
     [door.left, door.top, door.none, door.right, door.none, door.wheel, door.none, door.none, door.none],
     [NullTile, door.none, door.none, door.none, door.none, door.none, door.left, door.top, door.right, door.top]
 ]);
+  /* I know this return doesnot work but leaving it here to be solved in #161 */
+  /* Forest specific levels */
+  return new Level([
```

Comment:
I think, here we can use a new function body. This code is not reachable, right? I guess so.

Reply:
We would be fixing this in #160

Reply:
Sorry #161